### ISSUE TITLE
Dependency conflict when using rspec-core 3.7.0

### ISSUE THREAD

**Issue by samdturner** [mbj/mutant#736]
From `mutant/mutant-rspec.gemspec`, it appears that `mutant` is dependent on `rspec-core '>= 3.4.0', '< 3.8.0'`:
```
gem.add_runtime_dependency('rspec-core', '>= 3.4.0', '< 3.8.0')
```

We are using `rspec-core 3.7.0`, so I would expect that `mutant` would have not trouble depending on this version. However, bundler complains that there is a dependency conflict .

In my `Gemfile`, I have:
```
gem 'mutant-rspec',               '~> 0.8.14'
```

But when I run `bundle install`, I get:
```
$ bundle install
Bundler could not find compatible versions for gem "rspec-core":
  In snapshot (Gemfile.lock):
    rspec-core (= 3.7.0)

  In Gemfile:
    mutant-rspec was resolved to 0.8.14, which depends on
      rspec-core (< 3.7.0, >= 3.4.0)

    rspec-rails was resolved to 3.7.2, which depends on
      rspec-core (~> 3.7.0)

    rspec-retry was resolved to 0.5.6, which depends on
      rspec-core (< 3.8, > 3.3)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by spikeheap** [comment]
This looks like it's resolved for RSpec 3.7 in 0.8.15.

**Comment by mbj** [comment]
I'm closing this for now @samdturner if you have still the same problem, reopen and post here. My GH inbox is super busy from commercial activity I rarely see OSS popping up.
